app/views/users/dashboard.html.erb

```diff
@@ -3,39 +3,63 @@
 <div class="container justify-content-around">
     <div class="row"> 
         <div class="col-9">
-            <table class="table table-hover">
-                <thead>
-                    <tr>
-                        <th scope="col">#</th>
-                        <th scope="col">#</th>
-                        <th scope="col">#</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <th scope="row">1</th>
-                        <td>#</td>
-                        <td>#</td>
-                    </tr>
-                    <tr>
-                        <th scope="row">2</th>
-                        <td>#</td>
-                        <td>#</td>
-                    </tr>
-                </tbody>
-            </table>
+            <h3>activities</h3>
+                <% @activities.each do |activity| %>
+                  <div class="media border-top p-2">
+                    <div class="media-body row">
+                      <div class="img col-md-2">
+                        <% if User.find(activity.user_id).image? %>
+                            <%= image_tag(User.find(activity.user_id).image.url) %>
+                        <% else %>
+                            <%= image_tag('default.jpg') %>
+                        <% end %>    
+                      </div>
+                      <div class="content col-md-10"> 
+                        <% if  activity.action.is_a?(Lesson) %>
+                          <% lesson = activity.action %>
+                          <h5>
+                            <%= link_to lesson.user.name, user_path(lesson.user) %> learned <%= lesson.choices.pluck("correct").count(true) %> of <%= lesson.choices.pluck("content").count %> in <%= link_to lesson.category.title, category_path(lesson.category) %>
+                          </h5>
+                        <% elsif activity.action.is_a?(Relationship) %>  
+                          <% relationship = activity.action %>
+                          <h5>
+                            <%= link_to User.find(relationship.follower_id).name, user_path(current_user) %> followed <%= link_to User.find(relationship.followed_id).name, user_path(User.find(relationship.followed_id)) %>
```

Comment:
@YuheiTsunoda Improve this section with:
`<%= link_to activity.user.name, activity.user %> followed <%= link_to relationship.followed.name, relationship.followed %>`

In Relationship mode:
```
  belongs_to :follower, class_name: "User"
  belongs_to :followed, class_name: "User"
```